pkg/backup/backupsink/sink_utils.go

```diff
@@ -35,6 +38,21 @@ func ElidedPrefix(key roachpb.Key, mode execinfrapb.ElidePrefix) ([]byte, error)
 	return nil, nil
 }
 
+func elideMVCCKeyPrefix(
+	key storage.MVCCKey, mode execinfrapb.ElidePrefix,
+) (storage.MVCCKey, []byte, error) {
+	prefix, err := ElidedPrefix(key.Key, mode)
```

Comment:
nit: you can call `ElidedPrefix()` on each `Reset()` -- no need to update the cached version on each WriteKey. Perhaps then move this `elideMVCCKeyPrefix` into the sink key writer